### PR TITLE
Added Gayatri Mantra

### PR DESCRIPTION
**Pull request by gantanikhilraj**
Added Gayatri Mantra in data.json file :
(id, title, singer, source, image, audioUrl ),

Added Lyrics in lyrics.json file(id, lyrics).

Closes issue #1

### CODE REVIEW

File: data/data.json [hunk 1]
```diff
@@ -15,6 +15,14 @@
...
       "image": "https://hindipath.com/wp-content/uploads/2022/11/shriman-narayan-narayan-hari-hari-lyrics.jpg",
       "audioUrl": "https://drive.google.com/file/d/1u2ShDbbCVpQViwZW4_i-3lCNiQm_C9zh"
+    },
+    {
+      "id": "2a",
+      "title": "Gayatri Mantra",
+      "singer": "Anuradha Paudwal, Kavita Paudwal",
+      "source": "https://www.youtube.com/watch?v=nwRoHC83wx0",
+      "image": "https://images.bhaskarassets.com/webp/thumb/512x0/web2images/521/2020/01/10/gayatri-mantra_1578660625.jpg",
+      "audioUrl": "https://drive.google.com/file/d/1EKhmTX5olkRT88yVUa3B1_qmPnmqqfPd/view"
```
Comment (AmanNegi) [inline data/data.json]: Could you remove the `/view` from the link as it's not working as expected.

**Review by AmanNegi — COMMENTED** [review]
Could you please make the requested changes.

**Comment by gantanikhilraj** [comment]
> Could you please make the requested changes.

Yes, done @AmanNegi

**Review by AmanNegi — APPROVED** [review]
LGTM

**Pull request merged**